### ISSUE TITLE
what's meaning of "prediction confidence"？

### ISSUE THREAD

**Issue by BohanSi** [bowang-lab/scGPT#169]
I see the issue#85,  and I have the same question. In the paper "Attention mask for generative pretraining" part,  you said “scGPT selects the top 1/K genes from the unknown set with the highest prediction confidence to be included as known genes in the next iteration i + 1.” I'd like to know how to calculate the prediction confidence of generated genes. 
And I see the Supplementary Fig. 1b, does it mean the highest attention score among the unknown genes?
